### PR TITLE
Update to OMT schema v3.13.1

### PR DESCRIPTION
**Pull request by jsanz**
Replaces #6 

DO NOT MERGE BEFORE REMOVING d6df8b697e50dbf8532dfbe804bd817a80e199bf

The current styles are designed for OMT schema 3.12.1. Upon upgrading our data to OMT schema 3.13.1 we noticed some changes that warrant changes to the styles to avoid drastic changes to the end product. 

Any changes in this branch should be committed separately to make it easier to revert unwanted changes.

Use https://elastic.github.io/ems-basemap-editor?style=https://raw.githubusercontent.com/jsanz/osm-bright-desaturated-gl-style/omt-schema-v3.13.1/style.json to test the style in this PR.

Use https://elastic.github.io/ems-basemap-editor/comparator.html?style=osm-bright-desaturated&github=jsanz/osm-bright-desaturated-gl-style&ref=omt-schema-v3.13.1#2.25/48.44/52.93 to compare against production styles

### CODE REVIEW

File: layers/place/style.json [hunk 1]
```diff
@@ -81,6 +84,207 @@
...
+          "top-right",
+          "right",
+          "bottom-right",
+          "bottom",
+          "bottom-left",
+          "left",
+          "top-left",
+          "top"
+        ],
+        "text-radial-offset": 0.1,
```
Comment (nickpeihl) [inline layers/place/style.json]: IMO, it's better to center the text label on the label node for state/provinces which is the default if we remove these properties. But I'm open to discussing.

```suggestion
```

File: layers/place/style.json [hunk 2]
```diff
@@ -81,6 +84,207 @@
...
```
Comment (nickpeihl) [inline layers/place/style.json]: See other comment below.
```suggestion
```

File: layers/place/style.json (hunk 1 above)
Comment (jsanz) [reply layers/place/style.json]: This makes the labels to appear less often, but it also makes them _anchored_ and not _dancing_ around their center to accommodate to the other labels around which I accept is a bit annoying and confusing. So ok, let's remove this setting for both layers, thanks!

I'll update the generated style.json in a new commit.

**Review by nickpeihl — APPROVED** [review]
lgtm!

**Pull request merged**